ams/optimizer/common/src/main/java/com/netease/arctic/optimizer/job/common/Optimizer.java

```diff
@@ -1,4 +1,4 @@
-package com.netease.arctic.optimizer;
+package com.netease.arctic.optimizer.job.common;
```

Comment:
Package `com.netease.arctic.optimizer.job.common` seems to be too long for me, `com.netease.arctic.optimizer.job.common` is good enough for me.